i2p/src/main/java/bisq/i2p/I2pClient.java

```diff
@@ -177,6 +190,75 @@ private I2PSocketManager maybeCreateClientSession(String sessionId) {
         return sessionMap.get(sessionId);
     }
 
+    private Properties getPropertiesForEmbeddedRouter() throws IOException {
+        Properties p = new Properties();
```

Comment:
I would prefer to use "properties" instead of one char variable names. Improves code readability....